### ISSUE TITLE
Build not working with xbuild

### ISSUE THREAD

**Issue by JeroMiya** [OSVR/Managed-OSVR#28]
As reported here:
https://github.com/OSVR/OSVR-Unity/issues/8

There currently isn't a way to build Managed-OSVR on linux with xbuild.

**Comment by rpavlik** [comment]
cc @ChristophHaag

Looks like xbuild doesn't like the MSBuild foolishness required to be able to build both a .NET 2.0 and .NET 4.5 version.  Options:
- Use Windows-compiled assemblies: should work fine, I think, since it works fine for Android, though it's not really a solution to the "can't build except on Windows" problem
- Use newly cross-platform MSBuild open source project - not sure how this works on Linux ATM, but I saw a blog post about it.
- Create alternate xbuild files.

**Comment by ChristophHaag** [comment]
Yes, the windows compiled binaries work, but if people want to experiment/hack on it, they should be able to compile it. :)

Anyway, it seems to work with msbuild.

msbuild git doesn't build with current mono, I think it's fixed in mono git, but I haven't tested it.

But there is a prebuilt binary from corefx that can be installed like this if you have nuget installed:
`nuget install Microsoft.Build.Mono.Debug -Version 14.1.0.0 -OutputDirectory "~/msbuild" -source "https://www.myget.org/F/dotnet-buildtools/"`

And then you can build Managed-OSVR with
`mono ~/msbuild/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/MSBuild.exe ClientKit/ClientKit.csproj /p:Configuration=Release20 /p:OutputPath=../Managed-OSVR-Unity /p:NativeOutputSuffix32=x86 /p:NativeOutputSuffix64=x86_64 /p:DefineConstants="NET20"`

`NET20` doesn't seem to be defined anywhere in the build files, but manually defining it with `/p:DefineConstants="NET20"` seems to do the trick.

Not sure where to get the current version number, but here: https://github.com/dotnet/corefx/blob/master/build.sh#L191
